source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
+Once an incident is ended, be sure to schedule a post-mortem meeting to iterate on the design of your playbook. Automatic posts can help expose a timeline of events for review, such as when a task is marked as complete or when the commander changes. You can export the messages from the incident channel and use them to build a post-mortem document. Identify bottlenecks by explaining gaps between task completion, and suggest changes to the playbook for the next incident.
 
-Once complete, incident channels can be exported using the channel export option for analysis. Teams can identify bottlenecks in the incident by seeing time gaps between when checklist items are completed and incorporating necessary changes into the playbook for next incident.
+Playbooks
+^^^^^^^^^
+
+.. contents::
+  :backlinks: top
+  :local:
 
 Creating a playbook
-^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~
 
 A playbook must be defined before starting an incident.
 
 1. Navigate to **Main Menu > Playbooks & Incidents**.
-2. Select a template, or **+ Create a Playbook** to start a new playbook from scratch.
-4. Name your playbook.
-5. Edit the **Default Stage**, defining one or more steps to be taken by members of the incident.
-   * Optionally use descriptions on steps to add additional context for members of the incident. Descriptions support a limited form of markdown, including text styling and hyperlinks.
-   * Optionally define a slash command with the step, simplifying the completion of steps in the incident.
-6. Configure whether the incident channel should be public or private within the team.
-7. Share this playbook with other members of the team to allow them to use the playbook to start an incident, as well as edit the contents.
-
-Editing a playbook
-^^^^^^^^^^^^^^^^^^
-
-You can edit a playbook at any time. However, the changes will only be applied to future incidents - not the active incidents, or incidents that previously used that playbook.
-
-Navigate to **Main Menu > Playbooks & Incidents** and select the playbook you'd like to edit. You can:
+2. Start a **Blank Playbook**, or use the built-in **Incident Management Playbook** as a template.
+3. Name your playbook and provide a description.
+4. Within the **Tasks** tab, customize the stages and tasks:
+   * Create and name new tasks to capture actions your team should take to resolve the incident.
+   * Create new stages to group tasks meant to be completed together.
+   * Drag and drop to reorganize stages and tasks.
+   * Optionally add task descriptions to give additional context to members of the incident channel. Descriptions support a limited form of markdown, including text styling and hyperlinks.
+   * Optionally add a slash command to the task that can be executed by members of the incident channel as needed.
+5. Within the **Preferences** tab, customize how status updates are communicated:
```

Comment:
Should there be a step here to select the **Preferences* tab ahead of the current step 4?